### DISCUSSION TITLE
x:Array equivelent in WinUI?

### DISCUSSION THREAD

**Discussion by sam-wheat** [microsoft/microsoft-ui-xaml#8174] [Q&A] [answered]
I need to use an array of items as shown below.  How do I do this using WinUI?

      <local:DropDownButton.ItemsSource>
          <x:Array Type="local:DimensionAction">
            <local:DimensionAction>SortAscending</local:DimensionAction>
            <local:DimensionAction>SortDescending</local:DimensionAction>
            <local:DimensionAction>Hide</local:DimensionAction>
          </x:Array>
      </local:DropDownButton.ItemsSource>

**Comment by MikeHillberg** [comment, marked as answer]
There's no `x:Array` support, so what I do for this case is make a helper:

```cs
    public class List : List<object>
    {
        public List() { }
    }
```
And then:

```xml
<local:DropDownButton.ItemsSource>
    <local:List>
        <local:DimensionAction>SortAscending</local:DimensionAction>
        <local:DimensionAction>SortDescending</local:DimensionAction>
        <local:DimensionAction>Hide</local:DimensionAction>
    </local:List>
</local:DropDownButton.ItemsSource>
```

**Reply by sam-wheat** [reply]
Thank you!
